assets/js/2pulumi.js

```diff
@@ -87,13 +87,13 @@ function addLanguageFile(language, fn, code) {
     addCopyButton($(`#${filediv}`));
 }
 
-var couldNotConvertError =`<div id="couldnt-convert-code" class="container mx-auto pt-8">
+var couldNotConvertError = `<div id="couldnt-convert-code" class="container mx-auto pt-8">
     <div class="text-center max-w-2xl mx-auto">
         <h3>Sorry, we couldn't convert your code.</h3><br>
         <p class="text-lg mt-0 mb-16">
             There may be a problem with the code you submitted, or it might use a feature the
             converter doesn't yet support. To work with an engineer to help with your evaluation, please
-            <a href="{{< relref "/about#contactus" >}})" class="link">contact us</a> or
+            <a href="/about#contact" class="link">contact us</a> or
```

Comment:
`relref` seemed to break on the page. Switching to the above seems to work. Not familiar enough with hugo to know why the former didn't work.